scripts/make_release_branch.py

```diff
@@ -1,15 +1,62 @@
-from common import get_patch_version, inc_patch_version, GRADLE_PROPERTIES
+import argparse
+import datetime
+import json
+import re
+from urllib.request import urlopen
+
+import github_connect
+from common import get_patch_version, inc_patch_version, GRADLE_PROPERTIES, env
 from git import git_command
+from make_next_milestone import RELEASE_MANAGER_RE
+
+
+def send_slack_message():
+    slack_ids = json.loads(args.slack_ids)
+
+    milestone = github_connect.get_current_milestone(repo=env("GITHUB_REPOSITORY"))
+
+    release_version = milestone["title"]
+    release_manager = re.search(RELEASE_MANAGER_RE, milestone['description']).group(1)
+    link = milestone['html_url']
+    date = datetime.datetime.strptime(milestone["due_on"], "%Y-%m-%dT%H:%M:%SZ").date()
+
+    slack_user = slack_ids.get(release_manager)
+    message = {
+        "release_version": release_version,
+        "link": link,
+        "date": f'%s.%s.%s' % (date.day, date.month, date.year),
```

Comment:
nit: if `date.day` or `date.month` < 10, date won't contain leading zeros. You can add them via `%02d` parameter or via `{number:02d}` for f-strings